README.md

```diff
@@ -28,6 +28,12 @@ npm install -g yo
 npm install -g generator-liferay-fragments
 ```
 
+In case you are using **Liferay 7.2 GA 1** or a previous version, you have to install version 1.1.0 of the CLI instead:
```

Comment:
When connecting to **Liferay DXP 7.2.0** or **Liferay Portal CE 7.2.0** please use version 1.1.0. Newer versions depend on a new authentication mechanism that was not ready when those were released.

Reply:
I wasn't sure it was clear enough, so I've tried rewording it. What do you think?
In any case, this makes me wonder a few things:
- What should users of 7.1.x do?
- What's the target audience of the latest versions of the Fragment Toolkit? Should we avoid releasing as the latest versions that depend on un-released versions of Liferay Portal/DXP.

Finally, I think it's important that we to stick with using the word "Toolkit" instead of CLI or generator, to avoid confusion.

Reply:
Yeah, I prefer your version, I was quite unsure about the version naming
Regarding the questions:

- I should look into this, but I think it should work using the version 1.1.0 for these cases
- I think it makes sense to synchronize releasing new functionalities with new portal version,

What do you think about adding a compatibility matrix like for different liferay versions?

Reply:
A compatibility matrix sounds like a great idea :)